_projects/04-sip-process-reloaded.md

```diff
@@ -11,7 +11,7 @@ active: true
 home: true
 description: "Set of processes involving the community and aiming at improving the Scala language."
 ---
-SLIP: Scala library improvement process
-Anything related to scala modules (scala-xml, scala-swing, pickling, etc)
 
-SIP: Scala improvement process
+A SIP (Scala Improvement Process) is a process for submitting changes to the Scala language. Its main motivation is to become the primary mechanism to propose, discuss and implement language changes. In this process, all changes to the language go through design documents, called Scala Improvement Proposals (SIPs), which are openly discussed by a committee and only upon reaching a consensus are accepted to be merged into the Scala compiler.
+
+The aim of the Scala Improvement Process is to apply the openness and collaboration that have shaped Scala’s documentation and implementation to the process of evolving the language. This document captures our guidelines, commitments and expectations regarding this process.
```

Comment:
Maybe work on these two sentences. Not so sure about "the openness and collaboration that have shaped Scala’s documentation and implementation"...